utils/main.js

```diff
@@ -162,6 +162,28 @@ var util = {
     }
 
     return obj;
+  },
+
+  getDefaultConfig: function() {
+    return {
+      quote: 'single'
+    };
+  },
+
+  getConfig: function(options) {
+    var defaultConfig = util.getDefaultConfig();
+    var out = {};
+    var keys1 = Object.keys(defaultConfig);
+    for (var i = 0, key1 = keys1[i]; i < keys1.length; i++) {
+      out[key1] = defaultConfig[key1];
+    }
+    if (options !== null && options !== undefined && typeof options === 'object' && !Array.isArray(options)) {
```

Comment:
`typeof options === 'object'` should cover the first two falsy checks (null and undefined)